packages/cli-repl/src/run.ts

```diff
@@ -228,7 +228,10 @@ async function main() {
       ...connectionInfo.driverOptions,
     });
   } catch (e: any) {
-    console.error(`${e?.name}: ${e?.message}`);
+    // for debugging
+    if (process.env.MONGOSH_DISPLAY_STARTUP_STACK_TRACE)
+      console.error(e?.stack);
+    else console.error(`${e?.name}: ${e?.message}`);
```

Comment:
This is just for us, it's pretty annoying to have the stack trace not showing up for errors occuring during startup

Reply:
I should probably already know this answer, but why don't we display the stack trace by default all the time?